### PR TITLE
A new interface struct, BodyParameters

### LINKED ISSUE #1035: Create interfacing structs for InsertCelestialAbsoluteCartesian and InsertCelestialJacobiKeplerian
Characteristics of the body and PQ.

### PR DESCRIPTION
**Pull request by pleroy**
Fix #1035.

### CODE REVIEW

File: ksp_plugin_test/interface_test.cpp [hunk 1]
```diff
@@ -263,15 +263,16 @@ TEST_F(InterfaceTest, InsertMassiveCelestialAbsoluteCartesian) {
                              1.2345e6 * SIUnit<GravitationalParameter>()),
                     Property(&MassiveBody::mean_radius,
                              666 * Kilo(Metre))))));
+  BodyParameters body_parameters = {"1.2345e6  m^3/s^2",
```
Comment (eggrobin) [inline ksp_plugin_test/interface_test.cpp]: const

File: ksp_plugin_test/interface_test.cpp [hunk 2]
```diff
@@ -303,15 +304,16 @@ TEST_F(InterfaceTest, InsertOblateCelestialAbsoluteCartesian) {
                                  Pow<3>(Kilo(Metre)) / Pow<2>(Second)),
                     Property(&MassiveBody::mean_radius,
                              666 * Kilo(Metre))))));
+  BodyParameters body_parameters = {"1.2345e6  km^3 / s^2",
```
Comment (eggrobin) [inline ksp_plugin_test/interface_test.cpp]: const

File: ksp_plugin_test/interface_test.cpp (hunk 1 above)
Comment (pleroy) [inline ksp_plugin_test/interface_test.cpp]: Done.

File: ksp_plugin_test/interface_test.cpp (hunk 2 above)
Comment (pleroy) [inline ksp_plugin_test/interface_test.cpp]: Done.

**Pull request merged**